docs/platform/howto/update-tax-status.rst

```diff
@@ -0,0 +1,17 @@
+Update your tax status
```

Comment:
I'd be good to start this article with a short statement on what the article covers and what the user can learn from this doc.